restful-api-client/src/main/java/io/testproject/addon/restfulapiclient/internal/ReporterHelper.java

```diff
@@ -56,7 +56,8 @@ else if (serverResponse.responseCode == Integer.parseInt(expectedResponseCode))
         // Build the report & Fail the test in case there is JSON path and an error has occurred in extracting the json path from the response body
         if (!jsonPath.isEmpty()) {
             if (serverResponse.jJsonParseErrorMsg == null) {
-                resultStr.append("The value of \"").append(jsonPath).append("\" is \"").append(serverResponse.jsonParseResult).append("\". ").append(LS);
+                resultStr.append(String.format("The value of {{%s}} is {{%s}}. %s", jsonPath, serverResponse.jsonParseResult, LS));
```

Comment:
Why mixing String formatter and String builder? 
```suggestion
                resultStr.append("The value of {{").append(jsonPath).append("}}").append(" is {{").append("}}. ").append(serverResponse.jsonParseResult, LS);
```

Reply:
Why not?
I think it's more elegant and easy to understand.